trackbasket_be/models/krogerservice.py

```diff
@@ -4,6 +4,11 @@
 import os
 import json
 
+proxyDict = {
+              "http"  : os.environ.get('FIXIE_URL', ''),
+              "https" : os.environ.get('FIXIE_URL', '')
+            }
```

Comment:
experiment using fixie url to set a static ip address and find solution to heroku irregularities